src/patients/new/NewPatientForm.tsx

```diff
@@ -37,6 +38,9 @@ const NewPatientForm = (props: Props) => {
   })
 
   const onSaveButtonClick = async () => {
+    if (!patient.givenName && !patient.familyName) {
```

Comment:
In order to support all different types of name formats, I think that we should require that ONE of these fields is present (doesn't matter which one). @fox1t @StefanoMiC @tehKapa @lauggh what do you all think here?

Reply:
I agree with you @jackcmeyer. For example we already found cases where familiy names are not used (Burkina Faso). I think that the best choice is to make this configurable: the admin can decide mandatory fields for "all" forms. What do you think?